app/controllers/v1/actions_controller.rb

```diff
@@ -1,19 +1,25 @@
 class V1::ActionsController < V1::BaseController
-
   def create
     activity = Activity.find_by(template_id: params[:template_id])
+
     action = Action.create(person: person_from_params, activity: activity)
     if action.valid?
       output = { success: true }
     else
-      output = { error: 'could not create action' }
+      output = { error: action.errors.full_messages.first }
```

Comment:
Why only the first one? Would it be better to do `errors.full_messages.join('. ') + '.'`?